kola/tests/docker/docker.go

```diff
@@ -64,6 +65,18 @@ func init() {
 
 		MinVersion: semver.Version{Major: 1192},
 	})
+	register.Register(&register.Test{
```

Comment:
We aren't (yet) running kola on arm but do we have a way to restrict this to amd64? If not this can go in but file a bug to fix that since we do need to start testing arm at some point here...